pkg/receive/handler.go

```diff
@@ -196,6 +227,27 @@ func NewHandler(logger log.Logger, o *Options) *Handler {
 		h.replicationFactor.Set(1)
 	}
 
+	h.configuredTenantLimit.Set(float64(o.MaxPerTenantLimit))
+	h.tenantCurrentSeriesMap = map[string]float64{}
+
+	if (h.receiverMode == RouterOnly || h.receiverMode == RouterIngestor) && h.options.MaxPerTenantLimit != 0 {
+		// Use specified HTTPConfig to make requests to meta-monitoring.
+		httpConfContentYaml, err := h.options.MetaMonitoringHttpClient.Content()
+		if err != nil {
+			level.Error(h.logger).Log("msg", "getting http client config", "err", err.Error())
+		}
+
+		httpClientConfig, err := httpconfig.NewClientConfigFromYAML(httpConfContentYaml)
+		if err != nil {
+			level.Error(h.logger).Log("msg", "parsing http config YAML", "err", err.Error())
+		}
+
+		h.metaMonitoringClient, err = httpconfig.NewHTTPClient(*httpClientConfig, "thanos-receive")
```

Comment:
```suggestion
		h.metaMonitoringClient, err = httpconfig.NewHTTPClient(*httpClientConfig, "meta-mon-for-limit")
```

I believe this is a client name, so let's make it explicit. The fact we scrape it from thanos-receive will be obvious with `job`